src/pages/coursesPage/CoursesPage.tsx

```diff
@@ -21,7 +22,11 @@ const CoursesPage: React.FC = () => {
 
   return (
     <>
-      <CoursesList isLoading={isLoading} courses={courses} />
+      {!isLoading ? (
+        <CoursesList isLoading={isLoading} courses={courses} />
```

Comment:
`isLoading={isLoading}` prop isLoading always will be false. In another case this component just will not be rendered. I think this prop is useless, or maybe I didn't catch the point.